src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        uint96 multiplier;
+    }
+	struct RangePayment {
+        // Strategies & relative weights of shares in the strategies
+        StrategyAndMultiplier strategiesAndMultlipliers;
+        IERC20 token;
+        uint256 amount;
+        uint256 startTimestamp;
+        // duration in seconds
+        uint256 duration;
+	}
+
+    struct DistributionRoot {
+        // timestamp after which the root can be claimed against
+        uint32 activatedAfter;
+        // merkle root of the distribution
+        bytes32 root;
+    }
+
+    struct ClaimsTreeMerkleLeaf {
+        IERC20 token; 
+        uint256 amount;
+        // Explicit recipient of the claim
+        address recipient;
+    }
+
+    struct PaymentMerkleClaim {
+        ClaimsTreeMerkleLeaf[] leaves;
+        // The index of the root in the list of roots
+        uint32 rootIndex;
+        // proof of the claimaint's account root in the Merkle tree
+        bytes accountTreeProof;
+        // The indices and proofs of the leafs in the claimaint's merkle tree for this root
+        uint32[] leafIndices;
+        bytes[] tokenTreeProofs;
+    }
+
+    /// EVENTS ///
+
+    event RangePaymentCreated(address indexed avs, RangePayment rangePayment);
+    event PaymentUpdaterSet(address indexed oldPaymentUpdater, address indexed newPaymentUpdater);
+    event ActivationDelaySet(uint32 oldActivationDelay, uint32 newActivationDelay);
+    event GlobalCommissionBipsSet(uint16 oldGlobalCommissionBips, uint16 newGlobalCommissionBips);
+    event RecipientSet(address indexed account, address indexed recipient);
+    event RootSubmitted(bytes32 root, uint32 paymentsCalculatedUntilTimestamp, uint32 activatedAfter);
+    event PaymentClaimed(ClaimsTreeMerkleLeaf leaf);
+
+    /// VIEW FUNCTIONS ///
+
+    /// @notice The address of the entity that can update the contract with new merkle roots
+    function paymentUpdater() external view returns (address);
+
+    // @notice The owner of this contract
+    function owner() external view returns (address);
```

Comment:
do we simply use Ownable from OZ for this?

Reply:
I certainly think so!
this doesn't have to go in the interface, just added it for "completeness" / since the interface refers to the owner elsewhere.